liveblog/liveblog
# [LBSD-2730] - Add scrolling to freetypes in the editor

### Purpose

This PR adds scrolling to freetypes in the editor in order to view all types easily.

### What has changed

- Added `max-height` and `overflow-y` properties to the element.

### Steps to test

1. Navigate to the Editor's interface.
2. Click "Choose Post Type". A popup should show with a list of all post type. The list should be scrollable.

### Screenshots

| Original  | New |
| ------------- | ------------- |
| ![image](https://github.com/liveblog/liveblog/assets/14116447/d2829722-025a-4dbf-b3ee-988f1082ea66) | ![image](https://github.com/liveblog/liveblog/assets/14116447/0203f6c9-aac3-406b-80d6-fd219115754a)   |



Resolves: [LBSD-2730](https://sofab.atlassian.net/browse/LBSD-2730)

[LBSD-2730]: https://sofab.atlassian.net/browse/LBSD-2730?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
